static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -25,6 +25,13 @@
         @include mq(desktop) {
             width: 100%;
         }
+
+        // hide * home links, which are duplicated on large screens
+        &:first-child {
```

Comment:
@stephanfowler what do you think?

I mean obviously doing this is easy and works so I might be overthinking this. I just worry that it might be easy to accidentally hide things on desktop if the lists change in the future.